### DISCUSSION TITLE
Gopro 12 wifi credentials aren't used if set up with qr code

### DISCUSSION THREAD

**Discussion by maximko** [gopro/labs#699] [Q&A]
I got Gopro 12 for live streaming purposes and immediately installed labs to be able to start/stop and setup stream without an app. Docs specify that Auto Upload feature should be enabled once but I can't enable it since gopro subscription isn't available in my country (and I wouldn't enable if it isn't free anyway). Nevertheless I applied qr with wifi credentials and tried to start stream but camera was showing "..... button to cancel" indefinitely without switching to live mode. I thought pairing with an app would fix that. So after pairing nothing has changed and I decided to try to setup and start the stream within the app. That worked out and I was able to control stream with qr. Later I found that wifi credentials qr doesn't work dispite showing "approved" messge. The camera always connects to the network I provided within the app. Is it related to never enabled Auto Upload? Any workarounds? Gopro 12 and labs v2.00.70

**Comment by maximko** [comment]
I also found that camera doesn't reconnect if server closes connection

**Comment by dnewman-gpsw** [comment]
Once I'm back from break, I plan to look into livestream from Labs under HERO12, as it seems to be finicky, even with a subscription. It shouldn't require a subscription, just an App authentication, but I will have to confirm. I did have one HERO12 that livestream in seconds perfectly, but a second that behaves like you describe. HERO12 will always use the WiFi set up with the App, not the Labs JOIN credentials.  If you can Livestream with App, you should be about to use an RTMP target through Labs.  For my seconds camera I had to do factory reset, re-pair with Quik, do Livestream with Quik, now is working with Labs using !GLC command to go live.
